templates/app/new.go

```diff
@@ -12,7 +12,7 @@ import (
 // New ...
 func New(opts *Options) (*genny.Generator, error) {
 	g := genny.New()
-	if err := g.Box(packr.New("app/templates", "./templates")); err != nil {
+	if err := g.Box(packr.New("app/templates", "./templates/app/templates")); err != nil {
```

Comment:
```suggestion
	if err := g.Box(packr.New("app/templates", "./templates")); err != nil {
```

Reply:
`./templates/` wasn't working for me, does it work for you? Can you please kill if any zombie processes are running then run `app` and `serve` again?

Reply:
Checked multiple times: unless the path is `./templates` `starport app github.com/tendermint/blog` creates an empty directory, because packr doesn't pack anything, so there is nothing to serve. Can you check once again as well? Seem odd.

Reply:
reverted this again but the current behavior is, it only works when starport command is executed outside of the source code of starport. this is fine for developers but needs to fixed in future. Maybe we can replace `packr` with something lightweight